temporian/core/operators/arithmetic_scalar/less.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Equal scalar operator class and public API function definition."""
+
+from typing import Union, List
+
+from temporian.core import operator_lib
+from temporian.core.data.dtype import DType
+from temporian.core.data.event import Event
+from temporian.core.data.feature import Feature
+from temporian.core.operators.arithmetic_scalar.base import (
+    BaseArithmeticScalarOperator,
+)
+
+
+class LessScalarOperator(BaseArithmeticScalarOperator):

```

Comment:
This would also work :).
For some reasons, numpy uses "greater" and "less". I propose we keep the same.

*Keep less. Added TODO with question. Feel free to change if you prefer.